index.js

```diff
@@ -494,13 +537,54 @@ function onStream (client, onerror) {
     }
 
     // All requests to the APM Server must start with a metadata object
```

Comment:
nit: This comment was meant to be on the `stream.write(...)` below.

Reply:
✅ fixed in cb4d6e5f2ffe14d1e984b87ddf6be486a8590c14